imports/client/components/NotificationCenter.tsx

```diff
@@ -52,7 +54,7 @@ const StyledNotificationActionBar = styled.ul`
   list-style-type: none;
   margin: 0;
   padding: 0;
-  flex-direction: row;
+  flex-flow: wrap row;
```

Comment:
More width (see above) helps on the desktop, but some phone screens are still too small to fit them all on one row, so we need to be willing to wrap.